docs/topics/competitive-programming.md

```diff
@@ -70,36 +70,80 @@ hash-based maps and sets (`HashMap`/`HashSet`), tree-based ordered maps and sets
 Using a hash-set of integers to track values that were already reached while applying function `f`, 
 the straightforward imperative version of a solution to the problem can be written as shown below:
 
+<tabs>
+<tab title="Kotlin 1.6.0 and later">
```

Comment:
Add the tab synchronization, please (see as it has done for the build properties)

Reply:
Don't forget to add a separate group for it